millerg3_task2code.py

```diff
@@ -0,0 +1,120 @@
+# Gavin Miller, millerg3
+# https://docs.google.com/document/d/1ubqCXVn_pfDmnt3cz7FQ8piWil1egG5pBuyEQEn9wbA/edit?tab=t.0
+
+# Set up turtle and it's methods
+
+import turtle
+screen = turtle.Screen()
+screen.bgcolor("pink")
+alex = turtle.Turtle()
+print(alex.pos())
+alex.speed(0)
```

Comment:
There should not be any code outside of functions. Move this to the beginning of your `main` function.